components/main_menu/main_menu.jsx

```diff
@@ -182,6 +185,23 @@ class MainMenu extends React.PureComponent {
                 id={this.props.id}
                 ariaLabel={formatMessage({id: 'navbar_dropdown.menuAriaLabel', defaultMessage: 'main menu'})}
             >
+                {isFreeTrial &&
+                    <Menu.Group>
+                        <SystemPermissionGate permissions={Permissions.SYSCONSOLE_READ_PERMISSIONS}>
```

Comment:
We have a specific permission for the billing section in the System console with `SYSCONSOLE_WRITE_BILLING`
```suggestion
                        <SystemPermissionGate permissions={Permissions.SYSCONSOLE_WRITE_BILLING}>
```